doc/gallery/introduction/pytensor_intro.ipynb

```diff
@@ -0,0 +1,3311 @@
+{
```

Comment:
<p>Needs an authors section and the citation footer</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pytensor/pull/1585/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='111'/>

Reply:
Added 👍